app/helpers/application_helper.rb

```diff
@@ -22,4 +22,11 @@ def render_flash_message(flash_value)
     end
   end
 
+  def user_has_membership_application?
+    current_user.try(:membership_applications).try(:any?) && current_user.try(:membership_applications).try(:last).try(:persisted?)
+  end
+
+  def member_has_company?
+    current_user.is_member && !current_user.membership_applications.last.company.nil?
+  end
```

Comment:
It's being decided and refined and implemented as we speak.  :-)
Right now, it goes like this:
 When a user fills out an application, they have to include a Organization number (= Swedish thing). 
If the membership application is accepted
  then we create a full Company object and associate it with the Membership application

If the membership application is rejected, there's no need to instantiate a company (and save it in the db).